npg_qc_viewer/t/40-controller-QcOutcomes.t

```diff
@@ -429,7 +428,64 @@ subtest 'conditionally get wh info about tags' => sub {
   ok($response->is_success, 'response received for updating a seq entity');
 };
 
-subtest 'Conditional update of run/lane status in tracking' => sub {
+subtest 'rules relaxation for non-Illumina data' => sub {
```

Comment:
Perhaps name which rule is relaxed? It's hard to see from the test data and fixture what is right/wrong.